rl_studio/agents/f1/train_followlane_dqn_f1_gazebo_tf.py

```diff
@@ -318,8 +330,9 @@ def main(self):
                 )
             # reducing exploration
             if epsilon > epsilon_min:
-                epsilon *= epsilon_discount
-        
+                # epsilon *= epsilon_discount
+                epsilon -= epsilon_decay
```

Comment:
Did you see a better exploration/explotation using epsilon in this way? I once wondered if there would be an advantage substracting instead of multiplying, but I discarded this as a key point, preferring the asyntotic behavior of the multiplication.